src/fight.cpp

```diff
@@ -3030,6 +3046,7 @@ int check_recoil(struct char_data *ch, struct obj_data *gun)
   struct obj_data *obj;
   int rnum, comp = 0;
   bool gasvent = FALSE;
```

Comment:
We can probably just drop this variable, I don't know why it's here.

Reply:
You're right, I saw it was set below, didn't see that the value was not consumed anywhere.